crates/python_extractor/src/lib.rs

```diff
@@ -71,7 +72,7 @@ lazy_static! {
     };
 }
 
-pub fn read_file_to_str(input_file: &PathBuf) -> Result<String, anyhow::Error> {
+pub fn read_file_to_str(input_file: &Path) -> Result<String, anyhow::Error> {
```

Comment:
`&PathBuf` is an anti-pattern, use `&Path` which is a reference to an immutable Path.